trustkit/src/main/java/com/datatheorem/android/trustkit/config/TrustKitConfiguration.java

```diff
@@ -1,51 +1,62 @@
 package com.datatheorem.android.trustkit.config;
 
 import android.content.Context;
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
+
+import com.datatheorem.android.trustkit.TrustKit;
+import com.datatheorem.android.trustkit.utils.TrustKitLog;
+
+import org.xmlpull.v1.XmlPullParser;
+import org.xmlpull.v1.XmlPullParserException;
+
 import java.io.IOException;
 import java.security.cert.Certificate;
 import java.security.cert.CertificateException;
 import java.util.HashSet;
 import java.util.Set;
-import org.xmlpull.v1.XmlPullParser;
-import org.xmlpull.v1.XmlPullParserException;
 
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
 
 public class TrustKitConfiguration {
 
     @NonNull private final Set<DomainPinningPolicy> domainPolicies;
+    @Nullable private final Set<Certificate> trustAnchorsCertificates;
 
     // For simplicity, this works slightly differently than Android N as we use shouldOverridePins
     // as a global setting instead of a per-<certificates> setting like Android N does
     private final boolean shouldOverridePins;
     @Nullable private final Set<Certificate> debugCaCertificates;
 
-
     public static TrustKitConfiguration fromXmlPolicy(
-            @NonNull Context context, @NonNull XmlPullParser parser
+        @NonNull Context context, @NonNull XmlPullParser parser
     ) throws CertificateException, XmlPullParserException, IOException {
         return TrustKitConfigurationParser.fromXmlPolicy(context, parser);
     }
 
-
     protected TrustKitConfiguration(@NonNull Set<DomainPinningPolicy> domainConfigSet) {
         this(domainConfigSet, false, null);
     }
 
     protected TrustKitConfiguration(
-            @NonNull Set<DomainPinningPolicy> domainConfigSet,
-            boolean shouldOverridePins,
-            @Nullable Set<Certificate> debugCaCerts
+        @NonNull Set<DomainPinningPolicy> domainConfigSet,
+        boolean shouldOverridePins,
+        @Nullable Set<Certificate> debugCaCerts
```

Comment:
Does it make sense to add handling for overriding invalid domain behaviour into this configuration instead of having it separately?